templates/pvc-assets.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if (not .Values.mastodon.s3.enabled) -}}
+{{- if and (not .Values.mastodon.s3.enabled) (not .Values.mastodon.persistence.assets.existingClaim)-}}
```

Comment:
Looks good to me, don't worry about the chart version, I will take care of that shortly (bundling other PRs with it)

But the above line (and the corresponding one in `pvc-system.yaml` is invalid. There needs to be a whitespace before the `-}}` at the end.